### PR TITLE
Docs: adds whats new article for the 6.1 release

### PR DESCRIPTION
**Pull request by daniellee**
The what's new docs article for the 6.1 beta1 release.

### CODE REVIEW

**Comment by hugohaggmark** [comment]
LGTM. nice part about the permissions!

File: docs/sources/guides/whats-new-in-v6-1.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+weight = -12
++++
+
+# What's New in Grafana v6.1
+
+## Highlights
+
+### Ad hoc Filtering for Prometheus
+
+{{< imgbox max-width="30%" img="/img/docs/v61/prometheus-ad-hoc.gif" caption="Ad-hoc filters variable for Prometheus" >}}
```
Comment (hugohaggmark) [inline docs/sources/guides/whats-new-in-v6-1.md]: This is added also?

Comment (daniellee) [reply docs/sources/guides/whats-new-in-v6-1.md]: The image you mean? Yes - https://github.com/grafana/grafana.org/commit/e5ca7f9479a37342ae07245895fc1cf919c51d6d

**Review by marefr — APPROVED** [review]
LGTM will maybe suggest some more information before stable.

**Pull request merged**